easybuild/tools/github.py

```diff
@@ -1313,6 +1313,59 @@ def merge_url(gh):
         print_warning("Review indicates this PR should not be merged (use -f/--force to do so anyway)")
 
 
+def det_labels(file_info, pr_target_repo):
+    """
+    Determine labels from file_info.
```

Comment:
`Determine labels for a pull request based on provided information on files changed by that pull request.`